includes/Modules/Sign_In_With_Google.php

```diff
@@ -152,9 +155,18 @@ public function handle_login_errors( $error ) {
 	/**
 	 * Cleans up when the module is deactivated.
 	 *
+	 * Persist the clientID on module disconnection, so it can be
+	 * reused if the module were to be reconnected.
+	 *
 	 * @since 1.137.0
 	 */
 	public function on_deactivation() {
+		$pre_deactivation_settings = $this->get_settings()->get();
+
+		if ( isset( $pre_deactivation_settings['clientID'] ) ) {
+			( new Existing_Client_ID( $this->options ) )->set( $pre_deactivation_settings['clientID'] );
```

Comment:
Let's make it to be a private property instead of ad-hoc instantiation in multiple places.